### PR TITLE
Fix some missing taxon fields in importing process

### PR DESCRIPTION
**Pull request by qifeng-bai**

### CODE REVIEW

File: grails-app/services/au/org/ala/specieslist/ColumnMatchingService.groovy [hunk 1]
```diff
@@ -119,7 +119,7 @@ class ColumnMatchingService implements GrailsConfigurationAware {
             }
         }
 
-        headerResponse = parseHeadersCamelCase(headerResponse)
+        //headerResponse = parseHeadersCamelCase(headerResponse)
```
Comment (charvolant) [inline grails-app/services/au/org/ala/specieslist/ColumnMatchingService.groovy:122]: Remove commented out code

File: grails-app/services/au/org/ala/specieslist/QueryService.groovy [hunk 2]
```diff
@@ -16,6 +16,7 @@
 package au.org.ala.specieslist
 
 import groovy.time.*
+import io.micrometer.core.instrument.binder.okhttp3.OkHttpMetricsEventListener
```
Comment (charvolant) [inline grails-app/services/au/org/ala/specieslist/QueryService.groovy:19]: Is this used?

File: grails-app/views/speciesListItem/list.gsp [hunk 3]
```diff
@@ -142,7 +142,15 @@
                                 row.find("td.imageUrl").html("");
                             } else {
                                 var bie = "<a href=${bieUrl}/species/" + guid + " title='click to view species page'>"+result.matchedName + "</a>"
-                                row.find("td.matchedName").html(bie)
+
+                                if (result?.matchedSpecies?.vernacularName)
+                                    bie +="<br/>" + result?.matchedSpecies.vernacularName
+                                if (result?.matchedSpecies?.scientificNameAuthorship)
+                                    bie += "<br/>Autheor(s):" + result?.matchedSpecies.scientificNameAuthorship
```
Comment (charvolant) [inline grails-app/views/speciesListItem/list.gsp:149]: Should be "Author(s)" and preferably i18n'd

**Review by charvolant — CHANGES_REQUESTED** [review]
A few minor niggles

File: grails-app/views/speciesListItem/list.gsp (hunk 3 above)
Comment (qifeng-bai) [reply grails-app/views/speciesListItem/list.gsp:149]: i18n added

Comment (charvolant) [reply grails-app/views/speciesListItem/list.gsp]: I'm still reading "Autheor(s)" rather than "Author(s)"?

**Review by charvolant — APPROVED** [review]

**Pull request closed without merging**